From 20a8876d569828dce9074e89ada3a1bfa2879dda Mon Sep 17 00:00:00 2001
From: Sven Nierlein <sven@nierlein.de>
Date: Tue, 6 Feb 2018 20:24:41 +0100
Subject: [PATCH] fix legend parameter

the second argument to GraphPanel it $legendShow, not type. Right now
legend was always set to 'graph' which is true and always displays
the legend, regardless of the url parameter.
---
 histou/grafana/graphpanel/graphpanelinfluxdb.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/histou/grafana/graphpanel/graphpanelinfluxdb.php b/histou/grafana/graphpanel/graphpanelinfluxdb.php
index 9159847..b3b74bb 100644
--- a/histou/grafana/graphpanel/graphpanelinfluxdb.php
+++ b/histou/grafana/graphpanel/graphpanelinfluxdb.php
@@ -29,7 +29,7 @@ class GraphPanelInfluxdb extends GraphPanel
     **/
     public function __construct($title, $legendShow = SHOW_LEGEND, $id = -1)
     {
-        parent::__construct($title, 'graph', $id);
+        parent::__construct($title, $legendShow, $id);
     }
 
     public function createTarget(array $filterTags = array(), $datasource = INFLUXDB_DB)
